### PR TITLE
Rename node command to system

### LINKED ISSUE #761: Rename `node` command to `system`
### Expected behavior
To match the SDK exposed commands, the term `node` in core-CLI should be renamed to `system`. 

The name of the commands such as `node:info` should be `system:node-info` and similarly the name of `node:metadata` should be `system:metadata`.

For reference, please see the corresponding terms in SDK CLI: [system](https://github.com/LiskHQ/lisk-sdk/tree/development/commander/src/bootstrapping/commands/system)

### Actual behavior
The name of the command is `node` however, the examples in the command's description show the term `system`. 
node-system

### Steps to reproduce
`lisk core node -h`
`lisk core node:info -h`
### Which version(s) does this affect? (Environment, OS, etc...)
`lisk core alpha 18`

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?

This PR resolves #761

### How was it solved?

- [x] Rename node command to system

### How was it tested?

- [x] Local

### CODE REVIEW

File: docs/commands/system.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ OPTIONS
...
 
 EXAMPLES
-  node:metadata
-  node:metadata --data-path ./lisk
+  system:metadata
+  system:metadata --data-path ./lisk
 ```
 
-_See code: [dist/commands/node/metadata.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/node/metadata.ts)_
+_See code: [dist/commands/node/metadata.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/system/metadata.ts)_
```
Comment (vardan10) [inline docs/commands/system.md:49]:
```suggestion
_See code: [dist/commands/system/metadata.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/system/metadata.ts)_
```

File: docs/commands/system.md [hunk 2]
```diff
@@ -21,19 +21,19 @@ OPTIONS
...
 
 EXAMPLES
-  node:info
-  node:info  --data-path ./lisk
+  system:info
+  system:info  --data-path ./lisk
 ```
 
-_See code: [dist/commands/node/info.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/node/info.ts)_
+_See code: [dist/commands/node/info.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/system/node-info.ts)_
```
Comment (vardan10) [inline docs/commands/system.md:28]:
```suggestion
_See code: [dist/commands/system/info.ts](https://github.com/LiskHQ/lisk-core/blob/v4.0.0-alpha.14/dist/commands/system/node-info.ts)_
```

**Comment by vardan10** [comment]
We also need to change readme 
https://github.com/LiskHQ/lisk-core/blob/761-rename-node-command-to-system/README.md?plain=1#L87
https://github.com/LiskHQ/lisk-core/blob/761-rename-node-command-to-system/README.md?plain=1#L209

File: docs/commands/system.md (hunk 2 above)
Comment (priojeetpriyom) [reply docs/commands/system.md]: Applied.

**Review by vardan10 — APPROVED** [review]

File: docs/commands/system.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+`lisk-core system`
+================
+
+Commands relating to Lisk Core node.
+
+* [`lisk-core system:node-info`](#lisk-core-nodeinfo)
+* [`lisk-core system:metadata`](#lisk-core-nodemetadata)
```
Comment (nagdahimanshu) [inline docs/commands/system.md:7]:
```suggestion
* [`lisk-core system:node-info`](#lisk-core-systemnode-info)
* [`lisk-core system:metadata`](#lisk-core-systemmetadata)
```

**Review by nagdahimanshu — APPROVED** [review]

File: docs/commands/system.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* [`lisk-core system:node-info`](#lisk-core-systemnode-info)
+* [`lisk-core system:metadata`](#lisk-core-systemmetadata)
+
+## `lisk-core system:node-info`
+
+Get node information from a running application.
+
+```
+USAGE
+  $ lisk-core system:info
```
Comment (ManuGowda) [inline docs/commands/system.md:15]: @priojeetpriyom shouldn't this be `src/commands/system/node-info.ts` `system:node-info`?

**Review by ManuGowda — CHANGES_REQUESTED** [review]
minor change

File: docs/commands/system.md (hunk 4 above)
Comment (priojeetpriyom) [reply docs/commands/system.md]: Applied.

**Pull request merged**